services/scabbard/libscabbard/src/store/scabbard_store/action.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018-2022 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::SystemTime;
+
+use splinter::service::ServiceId;
+
+use crate::store::scabbard_store::{message::Scabbard2pcMessage, ScabbardContext};
+
+#[allow(dead_code)]
+#[derive(Debug)]
+pub enum ScabbardConsensusAction {
+    Scabbard2pcConsensusAction(ConsensusAction),
+}
+
+#[allow(dead_code)]
+#[derive(Debug)]
+pub enum ConsensusAction {
```

Comment:
We should bump these into two_phase (or some such) submodule.

Reply:
Updated